src/main/java/DukeLevel.java.rej

```diff
@@ -0,0 +1,151 @@
+diff a/src/main/java/DukeLevel.java b/src/main/java/DukeLevel.java	(rejected hunks)
+@@ -1,3 +1,4 @@
++import java.io.*;
+ import java.util.ArrayList;
+ import java.util.Scanner;
+ 
+@@ -15,6 +16,18 @@
+ 
+ public class DukeLevel {
```

Comment:
This method feels a bit long to me. Maybe you can consider using helper methods or dividing this method into several smaller methods?